### DISCUSSION TITLE
LNM and FS2024

### DISCUSSION THREAD

**Discussion by Timwintle1979** [albar965/littlenavmap#1202] [Q&A]
Hi All,

I wondered if anybody might know?  MS/Asobo have announced that their new product will include POI's and over 80,000 helipad's.  Do we know yet if LNM will be able to parse this data from the BGL files supplied by MSFS?  I know it might be a bit early to say yet as they've only entered the dev-alpha phase this last week but if anybody happens to know, I'd be curious.

Thanks
Tim

**Comment by albar965** [comment]
Hi Tim,

> Do we know yet if LNM will be able to parse this data from the BGL files supplied by MSFS?

I don't know yet since I have no access to MSFS 2024 yet but I'll buy it at release.

Looking at the [SDK documentation](https://docs.flightsimulator.com/msfs2024/html/1_Introduction/Introduction.htm) it seems quite the same as 2020 with just the paths adapted. 

Regarding BGL: My experience in the past is that the structure received minor changes and additions only. Have to see.

Discussion also here at Avsim: https://www.avsim.com/forums/topic/652764-msfs-2024-and-lnm/
Quoting myself :upside_down_face: 

> I'm pretty sure that SimConnect hasn't changed since this would break a lot of applications and add-ons.
> This means that you should be able to use your MSFS 2020 scenery library database to plan and use LNM for flight following in MSFS 2024 in the meantime.

If anybody has more information feel free to add it to #1198


Alex

**Comment by dr1ss** [comment]
Hello, any news on this please ?

**Reply by albar965** [reply]
Sorry. I did not get any pre-release version and have to buy this like anybody else. So it takes time.

At the moment I'm not sure if I can even support MSFS 2024 at all, let alone how much effort it is.

**Comment by dr1ss** [comment]
You should ask Asobo for a copy of the game, they will probably give it to u since LNM is a very popular third party app used by msfs users, if you just wanna take a look u can also subscribe  for a month to microsoft gamepass, the game is included with that subscription and it's cheap. (sent u a little something on Paypal)
Anyways LNM seem to detect my airplane etc.. I don't think that side of thing needs any work, the only thing what will need work is how to load the scenerie, let's hope they did not change the file format just the location.
No rush man even Navigraph are still trying to figure things out.

Have a good day and keep up the good work.

**Comment by albar965** [comment]
Discussion and more information here: https://www.avsim.com/forums/topic/655517-little-navmap-support-for-msfs-2024/

**Comment by Timwintle1979** [comment]
Hey @albar965 I see you've managed to find a way of getting the streamed data out of MSFS2024, big congrats!  That can't have been easy!!  I know it won't be your focus as it matters more about the core functionalities to you but did you happen to see how, if or where Points of Interest (POI's) are surfaced within the streamed data?

**Reply by albar965** [reply]
> Hey @albar965 I see you've managed to find a way of getting the streamed data out of MSFS2024, big congrats! That can't have been easy!! I know it won't be your focus as it matters more about the core functionalities to you but did you happen to see how, if or where Points of Interest (POI's) are surfaced within the streamed data?

The SimConnect interface is limited (see latest LNM changelog) and POIs are not available. There is no way to read them now since even the BGL files are hidden.
